corehq/apps/accounting/migrations/0092_revert_application_error_report_priv.py

```diff
@@ -0,0 +1,58 @@
+from django.db import migrations
+
+from django_prbac.models import Grant, Role
+
+from corehq import privileges, toggles
+from corehq.apps.accounting.utils import (
+    get_all_roles_by_slug,
+    get_granted_privs_for_grantee,
+)
+from corehq.util.django_migrations import skip_on_fresh_install
+
+ENTERPRISE_PLAN_ROLE_SLUG = 'enterprise_plan_v0'
+
+
+@skip_on_fresh_install
+def _remove_privilege_from_plan(apps, schema_editor):
+    try:
+        Role.objects.get(slug=privileges.APPLICATION_ERROR_REPORT).delete()
```

Comment:
Has not this be removed in the other PR? 
`privileges.APPLICATION_ERROR_REPORT`

Reply:
The privilege variable for `APPLICATION_ERROR_REPORT` is still being kept [here](https://github.com/dimagi/commcare-hq/blob/cdd228d7871e83ec7c90bf8a5cd80eb1c97d5f28/corehq/privileges.py#L112) so that we can revert the migration if needed.

Reply:
I believe it would be redundant post the first PR itself? Why not just use the string directly here?

Reply:
The privilege string has been added to the migration file ([ea22ed8](https://github.com/dimagi/commcare-hq/pull/34515/commits/ea22ed8a67569a1707f0d1dba690bf2f99815b0f)) and the privilege has been completely removed in the code related PR ([3c52921](https://github.com/dimagi/commcare-hq/pull/34514/commits/3c52921a95ed6495db1eac2b417c5af987455f91)).

Reply:
Ah, I was wrong in my previous statement. The `APPLICATION_ERROR_REPORT` does actually get used outside of the reverse code for this migration. I had completely forgotten that we reference it in previous migrations for the feature. Removing it from privileges would cause those migrations to break. Given this, I'll be keeping the reference in `privileges` and will be reverting the two commits I made.

Reply:
oh right, cool, thank god for tests!